assets/sass/elements/_forms.scss

```diff
@@ -167,12 +158,45 @@ textarea.form-control {
 }
 
 textarea.form-control {
+  background-image: none;
+  // Remove margin under textarea in Chrome and FF
+  display: block;
   // Disable opacity and background image for Firefox
   opacity: 1;
-  background-image: none;
 }
-// scss-lint:enable QualifyingElement
 
+select.form-control {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+  background-color: $white;
+  background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAYCAYAAACbU/80AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAIpJREFUeNrEkckNgDAMBBfRkEt0ObRBBdsGXUDgmQfK4XhH2m8czQAAy27R3tsw4Qfe2x8uOO6oYLb6GlOor3GF+swURAOmUJ+RwtEJs9WvTGEYxBXqI1MQAZhCfUQKRzDMVj+TwrAIV6jvSUEkYAr1LSkcyTBb/V+KYfX7xAeusq3sLDtGH3kEGACPWIflNZfhRQAAAABJRU5ErkJggg==');
+  background-position: center right 10px;
+  background-repeat: no-repeat;
+  background-size: 10px;
+  border-radius: 0;
+  // Fix the height for Firefox
+  height: 38px;
```

Comment:
We should avoid using `height` as that will break things for people who increase the font size (not zoom in). If you cannot solve it with padding, using `min-height` should be fine.
This is how it currently looks with 200% increased font size (which is what the [WCAG success criterion 1.4.4](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-scale) requires):

<img width="521" alt="" src="https://user-images.githubusercontent.com/108893/28582334-e1487054-715c-11e7-8d3e-34b7f611942c.png">